R/get_nsu_paths.R

```diff
@@ -12,7 +12,7 @@
 get_nsu_path <- function(year, ...) {
   nsu_file_path <- get_file_path(
     directory = fs::path(get_slf_dir(), "NSU"),
-    file_name = glue::glue("All_CHIs_20{year}.rds"),
+    file_name = glue::glue("All_CHIs_20{year}.gz.parquet"),
```

Comment:
Why are you using `.gz.parquet` for the file extension rather than just `.parquet`?